src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.ts

```diff
@@ -2,6 +2,9 @@ import { Component, ElementRef, HostBinding, HostListener, Input, OnDestroy, OnI
 import { CommunicatorEntity, ImageryCommunicatorService, IMapInstanceChanged } from '@ansyn/imagery';
 import { filter, take, tap } from 'rxjs/operators';
 import { AnnotationsVisualizer } from '../../../annotations.visualizer';
+import { AutoSubscription, AutoSubscriptions } from 'auto-subscriptions';
+import { selectIsMinimalistViewMode } from '@ansyn/map-facade';
```

Comment:
This causes a problem, as we saw.
If you will still hide, and not close, the context menu, check with Asaf about moving the component  to map-facade